src/main/java/core/basesyntax/BuffWriter.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class BuffWriter {
+    private String fileToRead;
+    private String fileToWrite;
+
+    public void writeInFile() {
+        CreateReportMessage message = new CreateReportMessage();
```

Comment:
it would be better to have arguments like `String fileToWrite` and `String text`
`writeInFile` should only write data to the file, not creating report and do other things